app/(tabs)/menu.tsx

```diff
@@ -0,0 +1,42 @@
+import { StyleSheet, View } from 'react-native'
+
+import HeaderBar from '@/components/header-bar'
+import { ThemedSafeAreaView } from '@/components/themed-safe-area-view'
+import { ThemedText } from '@/components/themed-text'
+
+export default function MenuScreen() {
+  return (
+    <ThemedSafeAreaView style={styles.container}>
```

Comment:
I think here you should use the common.screenContainer we created that style to be reused in the root container of the screens